.gitignore

```diff
@@ -35,3 +35,11 @@ c/docker/netcode.io
 c/valgrind/netcode.io
 docs
 *.exe
+
+valgrind/*
+docker/*
```

Comment:
do we need the `/*`s?

Reply:
Without them it included the Dockerfile.